packages/admin/cms-admin/src/pages/pageTree/PageVisibility.tsx

```diff
@@ -95,17 +95,35 @@ const PageVisibility = ({ page }: PageVisibilityProps): React.ReactElement => {
         );
     };
 
+    const chipBackgroundColor =
+        page.visibility === "Published"
+            ? theme.palette.success.main
+            : page.visibility === "Unpublished"
+            ? theme.palette.grey[100]
+            : theme.palette.common.white;
+
     return (
         <>
-            <Button size="small" onClick={handleMenuOpen} startIcon={<PageVisibilityIcon visibility={page.visibility} />} color="info">
-                <FormattedMessage
-                    id="comet.pages.pages.page.visibility"
-                    defaultMessage="{visibility, select, visible {Published} hidden {Unpublished} archived {Archived} other {unknown}}"
-                    values={{
-                        visibility: page.visibility === "Published" ? "visible" : page.visibility === "Unpublished" ? "hidden" : "archived",
-                    }}
-                />
-            </Button>
+            <Chip
+                icon={<ChevronDown />}
+                label={
+                    <FormattedMessage
+                        id="comet.pages.pages.page.visibility"
+                        defaultMessage="{visibility, select, visible {Published} hidden {Unpublished} archived {Archived} other {unknown}}"
+                        values={{
+                            visibility: page.visibility === "Published" ? "visible" : page.visibility === "Unpublished" ? "hidden" : "archived",
+                        }}
+                    />
+                }
+                sx={{
+                    backgroundColor: chipBackgroundColor,
+                    "&:hover": {
+                        backgroundColor: chipBackgroundColor,
+                    },
+                }}
```

Comment:
This should be possible using the different Chip variants: https://storybook.comet-dxp.com/?path=/story/comet-admin-mui--chip

Reply:
yes sure but the colors in the design are not available there. A normal color string can also not be passed because it needs a contrastText color

Reply:
Could you please elaborate more? Every variant that's in your screencast is available in the Storybook 🤔

Reply:
<img width="279" alt="image" src="https://github.com/user-attachments/assets/d26056f8-14b7-4d14-935b-c3181ceccb56">

White doesn't exist. Default color + outlined variant is a transparent chip, which will result in a grey chip with the grey background as can be seen in the screenshot above. Therefore it is not possible to achieve the design with the default colors and variants.

Reply:
It should not be necessary to use custom styling for Admin components in most cases. Maybe we need to change to chip component to achieve that. Could you please discuss this with UX and @jamesricky?

Reply:
I already talked with @jamesricky about it.

Reply:
Transparent, instead of white seems to be the correct color, according to the designs I have:

<img width="1525" alt="Screenshot 2024-08-19 at 09 29 32" src="https://github.com/user-attachments/assets/d7f416e1-9418-4aee-aa0c-090ed594c990">

Reply:
True. Maybe the design has been changed in the meantime^^

Reply:
I updated the screencast in the description